internal/applier/helm.go

```diff
@@ -138,9 +147,8 @@ func (h *Helm) getReleaseState(cl helmclient.ActionInterface, ext *ocv1alpha1.Cl
 		desiredRelease, err := cl.Install(ext.GetName(), ext.Spec.InstallNamespace, chrt, values, func(i *action.Install) error {
 			i.DryRun = true
 			i.DryRunOption = "server"
-			i.Labels = labels
```

Comment:
Question for reviewers: Should the helm release secrets also have these labels? I assumed they weren't necessary with the release name being deterministic based on the ClusterExtension name

Reply:
Maybe not necessary; would they otherwise be getting merged with labels by the `postrenderer`? If so I can see how they might get into a funny looking state.

Reply:
I don't think so. I think these labels are _explicitly_ on the release secrets created by the Helm client.

My understanding is that the `postrenderer` would add the labels to the manifests only _within_ the Helm Release

Reply:
I would take ^ with a grain of salt though. I'm _assuming_ the release secret is not included in the set of release manifests, but I don't actually know for sure.

Reply:
I think @gallettilance is planning to add release secret labels to populate CSV annotations there.

Seems like we should be able to trace what labels we currently set on the release secret and make sure we understand their purpose to decide whether we can stop populating them . I think we put [these](https://github.com/operator-framework/operator-controller/blob/main/internal/labels/labels.go) labels there, and they are used in a variety of ways (setup performant informers, get the installed bundles)